fullstack-templates/saas/turbowatch.ts

```diff
@@ -0,0 +1,40 @@
+import { defineConfig } from 'turbowatch';
+
+export default defineConfig({
+    project: __dirname,
+    triggers: [
+        {
+            expression: [
+                'allof',
+                ['not', ['dirname', '.next']],
+                ['not', ['dirname', 'node_modules']],
+                ['not', ['dirname', 'dist']],
+                ['not', ['dirname', 'public']],
+                [
+                    'anyof',
+                    ['match', '*.ts', 'basename'],
+                    ['match', '*.tsx', 'basename'],
+                    ['match', '*.js', 'basename'],
+                    ['match', '*.jsx', 'basename'],
+                    ['match', '*.css', 'basename'],
+                ],
+            ],
+            name: 'frontend',
+            onChange: async ({ spawn }) => {
+                await spawn`next build`;
+            },
+        },
+        {
+            expression: [
+                'allof',
+                ['not', ['dirname', '.cargo']],
+                ['not', ['dirname', 'target']],
+                ['anyof', ['match', '*.rs', 'basename'], ['match', '*.toml', 'basename']],
+            ],
+            name: 'backend',
+            onChange: async ({ spawn }) => {
+                await spawn`cargo shuttle run --working-directory backend`;
```

Comment:
Setting the dir to backend should not be needed, since there is a workspace manifest in the root. Can you try without it and confirm that it works?

Reply:
I have removed the workspace directory, you are right as we are in a workspace this is unnecessary here